.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
Not sure to understand what you are talking about.

What is the relation between sudo and my composer cache + oauth for dist-install system?